yellowbrick/contrib/missing/dispersion.py

```diff
@@ -0,0 +1,226 @@
+        If classes is None and a y value is passed to fit then the classes
+        are selected from the target vector.
+
+    kwargs : dict
+        Keyword arguments that are passed to the base class and may influence
+        the visualization as defined in other Visualizers.
+
+    Attributes
+    ----------
+    features_ : np.array
+        The feature labels ranked according to their importance
+
+    classes_ : np.array
+        The class labels for each of the target values
+
+    Examples
+    --------
+
+    >>> from yellowbrick.contrib.missing import MissingValuesDispersion
+    >>> visualizer = MissingValuesDispersion()
+    >>> visualizer.fit(X, y=y)
+    >>> visualizer.poof()
+    """
+
+    def __init__(self, alpha=0.5, marker="|", classes=None, **kwargs):
+
+        super(MissingValuesDispersion, self).__init__(**kwargs)
+        self.alpha = alpha
+        self.marker = marker
+
+        self.classes_ = classes
+
+        # Convert to array if necessary to match estimator.classes_
+        if self.classes_ is not None:
+            self.classes_ = np.array(classes)
+
+        # Set up classifier score visualization properties
+        if self.classes_ is not None:
+            n_colors = len(self.classes_)
+        else:
+            n_colors = None
+
+        self.colors = color_palette(kwargs.pop('colors', None), n_colors)
+
+
+    def get_nan_locs(self, **kwargs):
+        """Gets the locations of nans in feature data and returns
+        the coordinates in the matrix
+        """
+        if np.issubdtype(self.X.dtype, np.string_) or np.issubdtype(self.X.dtype, np.unicode_):
+            mask = np.where( self.X == '' )
+            nan_matrix = np.zeros(self.X.shape)
+            nan_matrix[mask] = np.nan
+
+        else:
+            nan_matrix = self.X.astype(float)
+
+        if self.y is None:
+            return np.argwhere(np.isnan(nan_matrix))
+        else:
+            nan_locs = []
+            for target_value in np.unique(self.y):
+                indices = np.argwhere(self.y == target_value)
+                target_matrix = nan_matrix[indices.flatten()]
+                nan_target_locs = np.argwhere(np.isnan(target_matrix))
+                nan_locs.append((target_value, nan_target_locs))
+
+            return nan_locs
+
+    def draw(self, X, y, **kwargs):
+        """Called from the fit method, this method creates a scatter plot that
+        draws each instance as a class or target colored point, whose location
+        is determined by the feature data set.
+
+        If y is not None, then it draws a scatter plot where each class is in a
+        different color.
+        """
+        nan_locs = self.get_nan_locs()
+        if y is None:
+            x_, y_ = list(zip(*nan_locs))
+            self.ax.scatter(x_, y_, alpha=self.alpha, marker=self.marker, label=None)
+        else:
+            self.draw_multi_dispersion_chart(nan_locs)
+
+    def draw_multi_dispersion_chart(self, nan_locs):
```

Comment:
This is also a very cool feature - and I can see myself using it a lot for my own classification projects!